extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -1284,32 +1261,32 @@ export function getPasswordMismatchMessage(fieldName: string): string {
 
 export async function setModelValues(inputComponents: InputComponents, model: Model): Promise<void> {
 	await Promise.all(Object.keys(inputComponents).map(async key => {
-		const value = await getInputComponentValue(inputComponents, key);
+		const value = await getInputComponentValue(inputComponents[key]);
 		model.setPropertyValue(key, value);
 	}));
 }
 
-async function getInputComponentValue(inputComponents: InputComponents, key: string): Promise<string | undefined> {
-	const input = inputComponents[key].component;
+async function getInputComponentValue(inputComponentInfo: InputComponentInfo): Promise<string | number | undefined> {
```

Comment:
It would be good to make a type for the `string | number | undefined` since this is used in multiple places.

Reply:
Ping @ranasaria

Reply:
done

Reply:
changed to ValidationValueType